Anlab.Core/Models/OrderDetails.cs

```diff
@@ -29,6 +29,12 @@ public OrderDetails()
         public IList<string> AdditionalEmails { get; set; }
 
         public string Project { get; set; }
+
+        public string Grind { get; set; } //Soil and Plant

```

Comment:
@jSylvestre I think these should be booleans, since it's really a yes/no question.